<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BabelConversation</class>
 <widget class="QWidget" name="BabelConversation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>365</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>383</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="chatLineEdit"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="sendPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
	<bool>true</bool>
      </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="suspendPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Suspend</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/icon/Resources/pause_icon.png</normaloff>:/images/icon/Resources/pause_icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="callPushButton">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Call</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/icon/Resources/call_icon.png</normaloff>:/images/icon/Resources/call_icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chatLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sendPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chatLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>BabelConversation</receiver>
   <slot>textLengthDisableSend(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>callPushButton</sender>
   <signal>clicked()</signal>
   <receiver>BabelConversation</receiver>
   <slot>startCall()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>suspendPushButton</sender>
   <signal>clicked()</signal>
   <receiver>BabelConversation</receiver>
   <slot>suspendOrResume()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>textLengthDisableSend(QString)</slot>
  <slot>startCall()</slot>
  <slot>suspendOrResume()</slot>
 </slots>
</ui>
